src/plugin.json

```diff
@@ -47,22 +47,6 @@
       "addToNav": true,
       "defaultNav": false
     },
-    {
```

Comment:
We remove them from the menu, but no the code (yet). It'll be done in the next PR.